federatedscope/core/workers/client.py

```diff
@@ -225,12 +226,12 @@ def callback_funcs_for_model_para(self, message: Message):
             # A fragment of the shared secret
             state, content, timestamp = message.state, message.content, \
                                         message.timestamp
-            self.msg_buffer['train'][state].append(content)
+            self.msg_buffer[STAGE.TRAIN][state].append(content)
```

Comment:
I'm not sure whether the 'stage' is an appropriate key name, as the key of `msg_buffer` should be message type (although it is not now: `msg_buffer`['train'] contains model, `msg_buffer`['eval'] contains eval results).

Maybe we should use the true message type as the key of `msg_buffer`.